src/components/Trade/TradeInput.tsx

```diff
@@ -315,9 +351,20 @@ export const TradeInput = ({ history }: RouterProps) => {
               type='submit'
               size='lg'
               width='full'
-              colorScheme={error || (isValid && !hasValidTradeBalance && !action) ? 'red' : 'blue'}
+              colorScheme={
+                error || (isValid && (!hasEnoughBalanceForGas || !hasValidTradeBalance) && !action)
```

Comment:
Non-blocking, I would abstract the `(!hasEnoughBalanceForGas || !hasValidTradeBalance)` to a variable like `canTrade`, but as I'm writing this, I'm realizing it's not easy to come up with a name that unites these two conditions so I'm happy with it the way it is currently.